python/dlisio/plumbing/channel.py

```diff
@@ -130,18 +130,18 @@ def source(self):
 
     @property
     def index(self):
-        """The Channel this Channel is indexed against, if any. See
+        """mnemonic of the channel this Channel is indexed against, if any. See
```

Comment:
(M)nemonic
 And probably remove "if any". Now there is always something

Reply:
If any, as in "if there is a index _Channel_"